plugins/SitesManager/Controller.php

```diff
@@ -259,4 +263,21 @@ private function getActiveTabOnLoad($templateData)
 
         return $tabToDisplay;
     }
+
+    private function getInviteUserLink()
+    {
+        $request = \Piwik\Request::fromRequest();
+        $idSite = $request->getIntegerParameter('idSite', 0);
+        if (!$idSite || !Piwik::isUserHasAdminAccess($idSite)) {
+            return 'https://matomo.org/faq/general/manage-users/#imanadmin-creating-users';
+        }
+        $request = \Piwik\Request::fromRequest();
+        $idSite = $request->getIntegerParameter('idSite', 0);
```

Comment:
This is duplicate code.
```suggestion
        $request = \Piwik\Request::fromRequest();
        $idSite = $request->getIntegerParameter('idSite', 0);
```

Reply:
Thanks, removed